## CI: «Build and push» failure is unrelated to this PR

The Docker image builds and pushes successfully. The failure is in the post-build Trivy SBOM scanning step:

```
installing Trivy binary
aquasecurity/trivy info checking GitHub for tag 'v0.67.2'
aquasecurity/trivy info found version: 0.67.2 for v0.67.2/Linux/64bit
##[error]Process completed with exit code 1.
```

The `nais/platform-build-push-sign` action (pinned to `8be8359`) checks out Trivy's `main` branch install script but requests release `v0.67.2` — the script appears to fail during download. This would break any PR or push to this repo right now.

All code-related checks pass: tests, govulncheck, staticcheck, gosec, govet, deadcode, fmt, helm-lint.

--- a/.github/workflows/main.yaml
+++ b/.github/workflows/main.yaml
@@ -56,7 +56,7 @@ jobs:
       - uses: jdx/mise-action@146a28175021df8ca24f8ee1828cc2a60f980bd5 # ratchet:jdx/mise-action@v3
         with:
           cache: false
-      - uses: nais/platform-build-push-sign@8be8359cd90915318ee8ab5fbc8337d04937ae70 # ratchet:nais/platform-build-push-sign@main
+      - uses: nais/platform-build-push-sign@a16d89d06262f12e3468a20b9cc70f5317290bab # ratchet:nais/platform-build-push-sign@main
         id: build-push-sign
         with:
           name: ${{ env.NAME }}

--- a/.github/workflows/pr.yaml
+++ b/.github/workflows/pr.yaml
@@ -40,7 +40,7 @@ jobs:
           cache: false
       - id: version-info
         run: echo "go-version=$(mise current go)" >> $GITHUB_OUTPUT
-      - uses: nais/platform-build-push-sign@8be8359cd90915318ee8ab5fbc8337d04937ae70 # ratchet:nais/platform-build-push-sign@main
+      - uses: nais/platform-build-push-sign@a16d89d06262f12e3468a20b9cc70f5317290bab # ratchet:nais/platform-build-push-sign@main
         id: build-push-sign
         with:
           name: ${{ env.NAME }}

